superset-frontend/src/components/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -42,19 +42,19 @@ InteractiveProgressBar.argTypes = {
   status: {
     control: {
       type: 'select',
-      options: ['normal', 'success', 'exception', 'active'],
     },
+    options: ['normal', 'success', 'exception', 'active'],
   },
   strokeLinecap: {
     control: {
       type: 'select',
-      options: ['round', 'square'],
     },
+    options: ['round', 'square'],
```

Comment:
This one doesn't seem to work, for whatever reason.

Reply:
I think this controls if the progressbar end is round or square like this:

![image](https://github.com/apache/superset/assets/76224658/dfc3f64b-6353-4aac-8862-474a1483c48a)
![image](https://github.com/apache/superset/assets/76224658/29118214-fd4b-4346-bff4-71cef56ac200)

Reply:
Oooooooh.... I was looking at the outside endcaps, of the _entire_ bar. I didn't even notice it on the filled-in section :P